ISICbalance/BalanceView.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  BalanceView.swift
+//  ISICbalance
+//
+//  Created by Rostislav Babáček on 11/03/2019.
+//  Copyright © 2019 Rostislav Babáček. All rights reserved.
+//
+
+import UIKit
+
+class BalanceView: UIView {
+
+    weak var screenStackView: UIStackView?
+    weak var balanceLabel: UILabel?
+    weak var balanceTitle: UILabel?
+    weak var reloadButton: UIButton?
+    weak var accountButton: UIButton?
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        self.backgroundColor = UIColor.MyTheme.backgroundColor
+        
+        let stackView = UIStackView()
+        stackView.axis = .vertical
+        stackView.spacing = 60
+        self.addSubview(stackView)
+        
+        stackView.snp.makeConstraints { make in
+            make.centerX.centerY.equalToSuperview()
+        }
+        
+        let balanceTitle = UILabel()
+        balanceTitle.text = L10n.Balance.title
+        balanceTitle.textColor = UIColor.MyTheme.labelBlue
+        balanceTitle.textAlignment = .center
+        balanceTitle.font = UIFont.systemFont(ofSize: 18)
+        self.balanceTitle = balanceTitle
+        stackView.addArrangedSubview(balanceTitle)
+        
+        let balanceLabel = UILabel()
+        balanceLabel.text = "1357 Kč"
```

Comment:
Piš si k tomu TODO jinak ti to tam zůstane ☝️

Reply:
Používáte normálně jen //TODO nebo #warning("TODO")?

Reply:
používáme `// TODO:` ve spojení se Swiftlint, který má v sobě pravidlo, pomocí kterého se `// TODO:` komentář zobrazí jako warning

Reply:
Dobrá. Mám teď v tomto PR měnit všechny warningy na "// TODO:" a přidat Swiftlint, nebo to mám nechat na později? A u toho Swiftlintu máte nějak nastavené parametry? Já jsem ho použil v semestrálce a řval mi například u každé mezery mezi bloky kódu. Říkali nám na SI, že v kódu nemají být mezery, ale každý blok by měl být v samostatné metodě, jenže mi to přijde jako overkill.

Reply:
určitě to nech na později, už takhle je ten PR dost velký

Reply:
Ok. Přidám to pak do nové branche.